actors/builtin/multisig/multisig_actor.go

```diff
@@ -117,7 +118,7 @@ func (a Actor) Constructor(rt vmr.Runtime, params *ConstructorParams) *adt.Empty
 	if params.UnlockDuration != 0 {
 		st.InitialBalance = rt.Message().ValueReceived()
 		st.UnlockDuration = params.UnlockDuration
-		st.StartEpoch = rt.CurrEpoch()
+		st.StartEpoch = params.StartEpoch
```

Comment:
We're changing behaviour here where there's probably nontrivial external code already written. If, at epoch 100000, someone sets a vesting duration of 10 days but forgets to set StartEpoch, they probably don't intend everything to immediately vest. 

We should probably either
- default to current epoch if start epoch is zero, or
- require it to be >= the current epoch

We can't simply require start epoch to be nonzero since that's a valid value during genesis state construction.

@whyrusleeping do you have a preference?

If not, go with defaulting to current epoch.

Reply:
hrm... zero should be a valid value. I think if they forget to set the value and it defaults to zero, then they can just make a new multisig? No strong opinions here though

Reply:
Ok, no change required then.